### DISCUSSION TITLE
glade file

### DISCUSSION THREAD

**Discussion by shulamy1** [community/community#35157] [Other Feature Feedback, Questions, & Ideas] [answered]
how can i get the glade file of birdie-alarm-mobile ?
ezik

**Comment by jfullstackdev** [comment, marked as answer]
I'm not very sure if you are referring to these projects, they are open source

https://github.com/GNOME/glade

https://github.com/obastani/glade

**Comment by shulamy1** [comment]
i am sorry i didn't know the app.ui is the glade file

ezik
